flight/move_to_test.py

```diff
@@ -0,0 +1,94 @@
+"""
+Main driver code for moving drone to each waypoint
+"""
+
+import asyncio
+import goto
+from mavsdk import System
+import logging
+from typing import List
+import sys
+import extract_gps
```

Comment:
Same thing about local imports here. Also, you should separate local imports from normal imports with a blank line.